components/CampingCard.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Card } from "../styles/styles";
+
+const CampingCard = ({ trip }) => {
+  return (
+    <Card>
+      <h2>{trip.location}</h2>
+      <p>Date: {trip.date}</p>
+      <p>Friends: {trip.friends.join(", ")}</p>
+    </Card>
```

Comment:
if the camping card is the card itself, you don't need to call it a card at this point here again?